File: Domain/Entities/Events/EventInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations.Schema;

namespace Mappen.Domain.Entities.Events
{

	[Table("EventInfos")]
	public class EventInfo
	{
		public int Id { get; set; }
		public string Name { get; set; }
		public string Description { get; set; }
		public string Parking { get; set; }
		public string Website { get; set; }
		public string PhotoUrl { get; set; }
		public string Email { get; set; }
		public long Phone { get; set; }
		public EventCategory Category { get; set; }
		public List<string> Tags { get; set; }

		public EventInfo Sync(EventInfo newInfo)
		{
			Name = newInfo.Name;
			Description = newInfo.Description;
			Parking = newInfo.Parking;
			Website = newInfo.Website;
			Email = newInfo.Email;
			Phone = newInfo.Phone;
			Category = newInfo.Category;
			Tags = newInfo.Tags;

			return this;
		}
	}
}

File: Application/Mappen/Scripts/App/EventInfo.js
var EventInfo = function () {
    var _self = this;
    this.Name = ko.observable();
    this.Description = ko.observable();
    this.DisplayDescription = ko.computed(function () {
        var description = _self.Description();
        return description ? description.split(/\n+/) : [];
    });
    this.Parking = ko.observable();
    this.Website = ko.observable();
    this.Email = ko.observable();
    this.Phone = ko.observable();
    this.Category = ko.observable();
    this.Tags = ko.observableArray();
    this.PhotoUrl = ko.observable();
}

EventInfo.prototype = {
    ProcessJson: function (json) {
        this.Name(json.Name);
        this.Description(json.Description);
        this.Parking(json.Parking);
        this.Website(json.Website);
        this.Email(json.Email);
        this.Phone(json.Phone);
        if (json.Category)
            this.Category(json.Category.Name);
        this.Tags(json.Tags);
        this.PhotoUrl(json.PhotoUrl);
    }
}